feature-libs/quote/components/summary/seller-edit/quote-summary-seller-edit.component.service.ts

```diff
@@ -245,13 +210,26 @@ export class QuoteSummarySellerEditComponentService {
     return isError ? { wrongFormat: {} } : null;
   }
 
-  protected performValidationAccordingToMetaData(
+  /**
+   * Returns maximum number of decimal places for the percentage discount.
+   * The value is read from the configuration, refer to attribute quote/maximumDecimalsForPercentageDiscount.
+   * In case this configuration is not present, default value is 8
+   *
+   * @returns Maximum number of decimal places for percentage discount
+   */
+  protected retrieveMaxNumberOfDecimalPlaces() {
+    return (
+      this.quoteUIConfig?.quote?.maximumDecimalsForPercentageDiscount ??
+      this.maximumDecimalsForPercentageDiscountDefault
+    );
+  }
+
+  protected performValidationForPercentageValue(
     input: string,
     groupingSeparator: string,
-    decimalSeparator: string,
-    numberTotalPlaces: number
+    decimalSeparator: string
   ): boolean {
-    const numberDecimalPlaces = 2;
+    const numberDecimalPlaces = this.retrieveMaxNumberOfDecimalPlaces();
     const regexEscape = '\\';
     const search: RegExp = new RegExp(regexEscape + groupingSeparator, 'g');
     const woGrouping = input.replace(search, '');
```

Comment:
Could you explain to me what `woGrouping` means?

Reply:
stands for 'withoutGroupingSeparator', will rename